warp/tests/locality/test4_mgpu.py

```
@@ -0,0 +1,133 @@
+# Old way (kept for reference):
+# memptr = memory.malloc_managed(nbytes)
+# cupy_arr = cp.ndarray((nx, ny), dtype=dtype, memptr=memptr)
+# managedA = wp.from_dlpack(cupy_arr.toDlpack())
+#
+# memptrC = memory.malloc_managed(nbytes)
+# cupy_arrC = cp.ndarray((nx, ny), dtype=dtype, memptr=memptrC)
+# managedC = wp.from_dlpack(cupy_arrC.toDlpack())
+
+wp.launch(range_fill_kernel, dim=(nx, ny), outputs=[managedA], device="cuda:0", block_dim=32)
+
+for iter in range(5):
```

Ruff (0.14.3)

Loop control variable `iter` not used within loop body

(B007)